internal/image/resource_image_test.go

```diff
@@ -542,8 +542,8 @@ func testAccImage_sourceInstanceWithSnapshot(projectName, instanceName string) s
 resource "incus_project" "project1" {
   name = "%[1]s"
   config = {
-	"features.images"   = false
-	"features.profiles" = false
+  "features.images"   = false
+  "features.profiles" = false
```

Comment:
So at least in the Github preview, this looks worse than it was before.
The same is true for most similar changes in this PR.

Are we dealing with mixed tabs/spaces by any chance? If so, we should pick one or the other so that things render properly :)

Reply:
Yes, there is a mix of tabs and spaces in the terraform strings and I agree, we should settle on one way of writing.

From what I see, in general spaces have been used with some "accidentally" added tabs. I removed the tabs and replaced them with spaces.

In this particular case, I added too little spaces, I will fix this.

Reply:
I believe `tofu fmt` or `terraform fmt` usually do spaces so makes sense to standardize on that.